tests/integration/examples/test_notebooks_gpu.py

```diff
@@ -213,7 +213,7 @@ def test_xdeepfm_integration(
     [
         (
             "1m",
-            50000,
+            10000,
```

Comment:
The metrics will change too, won't they?

Reply:
they changed a little bit, but from 10k to 50k they didn't change too much and the variation is absorbed by the tolerance